two_factor/utils.py

```diff
@@ -15,6 +15,9 @@ def default_device(user):
 
 
 def backup_phones(user):
+    if (getattr(settings, 'TWO_FACTOR_CALL_GATEWAY', None) is None
+            and getattr(settings, 'TWO_FACTOR_SMS_GATEWAY', None) is None):
+        return PhoneDevice.objects.none()
     if not user or user.is_anonymous:
         return PhoneDevice.objects.none()
```

Comment:
Could this be one if statement or do you think that will make it difficult to read and parse what is happening?

Reply:
i rewrote this too. Let me know what you think.